lib/rules/disallow-comma-before-line-break.js

```diff
@@ -40,10 +40,35 @@ module.exports = function() {};
 module.exports.prototype = {
```

Comment:
Need to update docs.

And I'm not sure I understand what beforeComma option really means ;-( Perhaps there are better names.